systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -223,12 +229,18 @@ private static Kafka waitFor(Kafka kafka) {
             DeploymentUtils.waitForDeploymentReady(io.strimzi.api.kafka.model.KafkaResources.entityOperatorDeploymentName(name));
             LOGGER.info("Entity Operator pods are ready");
         }
-        // Kafka Exporter is not setup everytime
+        // Kafka Exporter is not setup every time
         if (kafka.getSpec().getKafkaExporter() != null) {
             LOGGER.info("Waiting for Kafka Exporter pods");
             DeploymentUtils.waitForDeploymentReady(KafkaExporterResources.deploymentName(name));
             LOGGER.info("Kafka Exporter pods are ready");
         }
+        // Cruise Control is not setup every time
+        if (kafka.getSpec().getCruiseControl() != null) {
+            LOGGER.info("Waiting for Cruise Control pods");
+            DeploymentUtils.waitForDeploymentReady(KafkaResources.cruiseControlDeploymentName(name));
```

Comment:
The deployment name can be grabbed from the CruiseControlResources [1]

[1] https://github.com/kyguy/strimzi-kafka-operator/blob/cruise-control-deployment/api/src/main/java/io/strimzi/api/kafka/model/CruiseControlResources.java#L13-L20